<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pyArchInit Mapper</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/H_check.png</normaloff>../icon/H_check.png</iconset>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="widget">
   <widget class="QTableWidget" name="template_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>256</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>254</number>
    </attribute>
    <column>
     <property name="text">
      <string>Template</string>
     </property>
    </column>
   </widget>
   <widget class="QTableView" name="data_table">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>311</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragOnly</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectColumns</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="mapping_table">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>431</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DropOnly</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>212</number>
    </attribute>
    <column>
     <property name="text">
      <string>Template</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data</string>
     </property>
    </column>
   </widget>
   <widget class="QToolButton" name="toolButton_load">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Load data</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_template">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>20</number>
    </property>
    <property name="placeholderText">
     <string>Choose a template for the mapping</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convert_data">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_mapping">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add mapping</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_mapping">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>360</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove mapping</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>1021</width>
      <height>192</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="update_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_data">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>660</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1044</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>&amp;help</string>
    </property>
   </widget>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
